src/sdk/account/utils/Utils.ts

```diff
@@ -391,3 +396,32 @@ export type EthersWallet = {
   address: Address | string
   provider: AnyData
 }
+
+export const getAllowance = async (
+  client: PublicClient,
+  accountAddress: Address,
+  tokenAddress: Address
+): Promise<bigint> => {
+  const approval = await client.readContract({
+    address: tokenAddress,
+    abi: erc20Abi,
+    functionName: "allowance",
+    args: [accountAddress, BICONOMY_TOKEN_PAYMASTER]
+  })
+
+  return approval as bigint
+}
+
+export const isNexusClient = (client: Client): client is NexusClient => {
+  return client.name === "Nexus Client"
+}
+
+export type ExtendedBundlerClient = BundlerClient & {
```

Comment:
hmmm, doesn't BundlerClient already have an optional paymasterContext? You can remove this I think?